src/servers/src/error.rs

```diff
@@ -637,6 +675,10 @@ impl ErrorExt for Error {
             | MissingQueryContext { .. }
             | MysqlValueConversion { .. }
             | UnexpectedPhysicalTable { .. }
+            | ParseJson { .. }
+            | ToStructuredLog { .. }
+            | UnsupportedContentType { .. }
+            | InsertLog { .. }
```

Comment:
I don't think `InsertLog` is in the `InvalidArguments` case, may be `StorageUnavailable`?

Reply:
I found that this error wasn't being used, so I deleted it.